homeassistant/components/somfy/cover.py

```diff
@@ -78,6 +114,20 @@ def current_cover_position(self):
             position = 100 - self.cover.get_position()
         return position
 
+    @property
+    def is_opening(self):
+        """Return if the cover is opening."""
+        if not self.optimistic:
```

Comment:
Why this condition?

Reply:
The device doesn't actually report back that its opening so we assuming that since our api call was accepted, it actually did something.